### ISSUE TITLE
MacOS builds broken

### ISSUE THREAD

**Issue by axelrindle** [axelrindle/vscode-duplicate#2]
MacOS tests are currently broken: https://github.com/axelrindle/vscode-duplicate/actions/runs/5416735548/jobs/9846857160

While this shouldn't affect actual usage, I have no Apple devices to test this for myself. Thus the MacOS CI is disabled indefinitely.

**Comment by backermanbd** [comment]
Does this mean we can no longer use this extension in MacOS?

**Comment by axelrindle** [comment]
No. This means the CI (build and test) is not run on a MacOS system. Thus I cannot guarantee the extension to be working on MacOS.
